recipes/hash-library/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "8.0":
```

Comment:
The upstream repository just refers to the versions as "v8" (and previous versions "v1" .. "v7"). I first tried to simply use "8" as version number but Conan requires the version to consist of at least two characters, so I went with "8.0". Another option could be to use "v8".

Reply:
Yes, that's correct.